File: core/src/ru/myitschool/dcrawler/ui/screens/AdvancedScreen.java
package ru.myitschool.dcrawler.ui.screens;

import com.badlogic.gdx.InputMultiplexer;
import com.badlogic.gdx.Screen;
import ru.myitschool.dcrawler.MyGdxGame;

/**
 * Created by Voyager on 03.04.2018.
 */
public abstract class AdvancedScreen implements Screen {

    private InputMultiplexer input;

    public AdvancedScreen() {
        input = new InputMultiplexer();
        MyGdxGame.setScreenInput(input);
    }

    public InputMultiplexer getInput() {
        return input;
    }
}

File: core/src/com/wolg_vlad/dcrawler/ui/screens/AdvancedScreen.kt
package com.wolg_vlad.dcrawler.ui.screens

import com.badlogic.gdx.InputMultiplexer
import com.badlogic.gdx.Screen
import com.wolg_vlad.dcrawler.MyGdxGame

/**
 * Created by Voyager on 03.04.2018.
 */
abstract class AdvancedScreen : Screen {
    val input: InputMultiplexer

    init {
        input = InputMultiplexer()
        MyGdxGame.setScreenInput(input)
    }
}